src/instrument.js

```diff
@@ -51,44 +65,62 @@ async function findInstruments(oreAccountName, activeOnly = true, category = und
   return rows
 }
 
-async function getInstruments(oreAccountName, category = undefined, filters = []) {
-  if (category) {
-    filters.push(function(row) {
-      return row["instrument"]["instrument_class"] === category
-    })
-  }
-
-  const rows = await getAllInstruments.bind(this)(oreAccountName, filters)
-  return rows
-}
-
-function getRight(instrument, rightName) {
-  const rights = instrument["instrument"]["rights"]
-  for (let i = 0; i < rights.length; i++) {
-    let right = rights[i]
-    if (right["right_name"] === rightName) {
-      return right
-    }
-  }
-}
-
 async function saveInstrument(oreAccountName, instrument) {
   // Confirms that issuer in Instrument matches signature of transaction
   // Creates an instrument token, populate with params, save to issuer account
   // Saves endpoints to endpoints_published
   let options = {authorization: `${oreAccountName}@active`}
   let contract = await this.eos.contract(APIM_CONTRACT_NAME, options)
-  instrument.start_time = instrument.start_time || Date.now()
-  instrument.end_time = instrument.end_time || Date.now() + ONE_YEAR
-  await contract.publishapi(oreAccountName, instrument.apiName, instrument.rights, instrument.description, instrument.start_time, instrument.end_time, options)
+  let start_time = Date.now()
+  let end_time = Date.now() + ONE_YEAR
```

Comment:
Any reason why we don't want to allow for custom start_time & end_time?

Reply:
we do..don't need it here. We can pass these when we create instruments.